apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -80,7 +80,12 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
                 ExtendedDocumentStatus.DRAFT,
                 ExtendedDocumentStatus.ALL,
               ].map((value) => (
-                <TabsTrigger key={value} className="min-w-[60px]" value={value} asChild>
+                <TabsTrigger
+                  key={value}
+                  className="min-w-[60px] hover:text-gray-100"
```

Comment:
We typically try to avoid using the tailwind colors instead opting for our css variable versions. In this case I'd suggest using `hover:text-primary` to match the active color for the tab 😄

Reply:
Thanks for the suggestion @Mythie. I've made the required changes 👍🏼